.github/workflows/closing-pr.js

```diff
@@ -0,0 +1,19 @@
+// https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue
+const CLOSING_SYNTAX_REGEX = /(?:close|fix|resolve)[sd]?\s+#(\d+)/i;
+const HAS_CLOSING_PR_LABEL = 'has-closing-pr';
+
+module.exports = async ({ context, github }) => {
+  const { body } = context.payload.pull_request;
+  // Find a closing syntax after removing comments
+  const match = body.replace(/<!--(.+?)-->/gs, '').match(CLOSING_SYNTAX_REGEX);
```

Comment:
The PR template contains `Close #123` as an example. `replace(/<!--(.+?)-->/gs, '')` removes it.